components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -329,6 +331,20 @@ function CustomLeague:_getMaps()
 	return maps
 end
 
+function CustomLeague:_checkMapInformation(name, link, game)
+	local data = mw.ext.LiquipediaDB.lpdb('datapoint', {
+		conditions = '[[type::map]] AND [[pagename::' .. link:gsub(' ', '_') .. ']]',
+		query = 'name, extradata'
+	})
+	if #data and Table.isNotEmpty(data[1]) then
```

Comment:
Is `#data` really what you're looking for here? You'll either get a truthy result or an error

Reply:
```suggestion
	if #data > 0 and Table.isNotEmpty(data[1]) then
```